tool/plugin/lib/plugin.dart

```diff
@@ -731,6 +705,7 @@ abstract class ProductCommand extends Command {
   Future<int> run() async {
     await _initGlobals();
     await _initSpecs();
+    _handleSymlinksOnWindows();
```

Comment:
This is the only necessary change for Windows currently.

Reply:
You can also create the local repo in such a way that symlinks work; see line 139 in CONTRIBUTING.md. But, now we don't have to remember to do that. Thanks!

Reply:
Oh sure that line can be removed too IMO.